src/firecracker/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "firecracker"
 version = "1.5.0-dev"
 authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
 edition = "2021"
-build = "../../build.rs"
+build = "build.rs"
```

Comment:
If we modify the top-level build.rs/, we would have to remember to modify this one as well?

Reply:
Since all we do is pass through `CARGO_PKG_VERSION`, maybe we should just rely on that and delete the toplevel build.rs? Though that is somewhat larger in scope.

Reply:
Ah, right, I messed this up slightly, good catch! There is only supposed to be one build script anymore, but I got distracted by the seccomp changes and forgot to follow through with that